ProyectoSemestral3/Assets/Scripts/InventoryScripts/ItemController.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ItemController : MonoBehaviour
+{
+    public Item item;
+
```

Comment:
Remove extra space and why public? also `_`, and initialize it.

Reply:
Done